### PR TITLE
Minor cleanup to remove dependencies on the old config code.

### PR DESCRIPTION
**Pull request by ozevren**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[X] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: galley/cmd/galley/cmd/server.go [hunk 1]
```diff
@@ -96,104 +96,104 @@ func serverCmd() *cobra.Command {
...
+	loggingOptions.AttachCobraFlags(svr)
+	_ = viper.BindPFlags(svr.PersistentFlags())
 
 	cobra.OnInitialize(setupAliases)
 
-	return serverCmd
+	return svr
 }
 
 func setupAliases() {
```
Comment (sushicw) [inline galley/cmd/galley/cmd/server.go]: While you're here, can you please make sure that the viper aliases are correctly all there? At a glance it looks like some of them are missing, e.g. enableAnalysis.

Comment (ozevren) [reply galley/cmd/galley/cmd/server.go]: Analysis is there:
https://github.com/istio/istio/blob/ff382fe4ceed2bd5bdd57ed1f7b5adf69c6a7566/galley/cmd/galley/cmd/server.go#L213

Comment (sushicw) [reply galley/cmd/galley/cmd/server.go]: Whoops, I picked the wrong example.

I'm not seeing useOldProcessor or enableServiceDiscovery, though.

**Pull request merged**